### PR TITLE
F.A.Q testing

### PR DESCRIPTION
**Pull request by alexevic**
I created full F.A.Q. page TA. Fixed some naming in vision TA.

### CODE REVIEW

File: src/features/faq.feature [hunk 1]
```diff
@@ -0,0 +1,176 @@
+Feature: Tests for FAQ page
+
+    Scenario: Check FAQ page title and description
+    Given the user is on the Faq page
+    Then the Faq title, header and description should be correct
```
Comment (Deimokas) [inline src/features/faq.feature]: Could be three steps, one for title, one for header and one for description

File: src/page-objects/faq.page.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        return getElement('[ta-id="faq_146432569"]');
+    }
+
+    async getAccordionItemDetails(nth) {
+        return {
+            image: await getNthElement('div.brighter-accordion-item__header__content > brighter-icon > svg > use' ,nth),
+            icon: await getNthElement('div.brighter-accordion-item__toggle.ng-star-inserted > brighter-accordion-item-toggle > brighter-icon > svg > use', nth),
+            title: await getNthElement('[ta-id="faq_201393979"]', nth),
+            description: await getNthElement('[ta-id="faq_166895400"]', nth),
+            item: await getNthElement('[ta-id="faq_217190828"]', nth),
```
Comment (Deimokas) [inline src/page-objects/faq.page.ts]: Refactor that only relevant methods would be used meaning, move out unused methods

**Review by Deimokas — CHANGES_REQUESTED** [review]

**Review by KietasRiestutelis — APPROVED** [review]

File: src/features/faq.feature (hunk 1 above)
Comment (alexevic) [reply src/features/faq.feature]: Test was rewritten, more steps added.

File: src/page-objects/faq.page.ts (hunk 2 above)
Comment (alexevic) [reply src/page-objects/faq.page.ts]: Function added to select only one element at the time. Now for test, code gets only elements that are used.

**Review by KietasRiestutelis — APPROVED** [review]

**Pull request merged**